trestle/utils/log.py

```diff
@@ -92,6 +94,16 @@ def set_log_level_from_args(args: argparse.Namespace) -> None:
         sys.excepthook = exception_handler
 
 
+def get_current_verbosity_level(logger: logging.Logger) -> int:
+    """Get the current verbosity level based on logging level."""
+    level = logger.getEffectiveLevel()
+    if level < logging.DEBUG:
```

Comment:
Do we want to distinguish between 0 i.e. `NONSET` and this `TRACE` level?

Reply:
I think 0 is not really used and DEBUG - 5 is a common value for below DEBUG level.  Made special get_trace_level() function to encapsulate this.